### PR TITLE
ci: update workflows

### PR DESCRIPTION
**Pull request by drupol**
This PR:

- Fix broken "Upgrade flake" action
- Bump action versions

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -43,8 +43,8 @@ jobs:
           major=${branch%%.*}
           minor=${branch#*.}
           attr=php$major$minor
-          echo "::set-output name=major::$major"
-          echo "::set-output name=attr::$attr"
+          echo "major=php$major$minor" >> $GITHUB_OUTPUT
```
Comment (jtojnar) [inline .github/workflows/build.yaml:46]: This should be just `$major`

Comment (jtojnar) [reply .github/workflows/build.yaml:46]: Also would be nice to link https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/ in the commit message.

Comment (drupol) [reply .github/workflows/build.yaml]: Oh nice catch!

I fixed it from my phone, feel free to rebase or I'll do it tomorrow

Comment (drupol) [reply .github/workflows/build.yaml]: Fixed!

**Review by jtojnar — APPROVED** [review]

**Pull request merged**